app/models/journeys/service_access_code.rb

```diff
@@ -0,0 +1,35 @@
+module Journeys
+  class ServiceAccessCode < ApplicationRecord
+    scope :for_journey, ->(journey) { where(journey: journey) }
+
+    scope :used, -> do
```

Comment:
It might be preferable to have a `used` column or similar on this model which is updated on claim submission, for speed and avoiding having model code tied to database schema (which sometimes gets overlooked).